cmd/generate.go

```diff
@@ -31,7 +31,7 @@ func getEnvs() map[string]string {
 	m := make(map[string]string)
 
 	for _, env := range envs {
-		results := strings.SplitN(env, "=", 1)
```

Comment:
a small unittest for this would be nice

Reply:
We already have a test for this, check out the failing test for main branch

Reply:
@dennislapchenko following up on this

Reply:
Seems like this should be done with Cut, as per the docs.

https://pkg.go.dev/strings#SplitN

```To split around the first instance of a separator, see Cut.```


Perhaps something like:
```diff
- results := strings.SplitN(env, "=", 2)
- m[results[0]] = results[1]
+ varName, varValue, found := strings.Cut(env, "=")
+   if found {
+     m[varName] = m[varValue]
+   }

Reply:
either way works, let's just go with what's already in the code and passing tests